src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  if (days < 3) {
+    rentalCost = days * 40;
+  } else if (days < 7) {
+    rentalCost = days * 40 - 20;
+  } else {
+    rentalCost = days * 40 - 50;
```

Comment:
You should replace the literals in the expressions with constants to avoid hardcoding. For instance, instead of `40`, `20`, and `50`, define constants such as `DAILY_RENTAL_COST`, `SHORT_TERM_DISCOUNT`, and `LONG_TERM_DISCOUNT` respectively.